### PR TITLE
Add Contextual Helpers

### PR DESCRIPTION
**Pull request by lookacat**
## Description
See https://github.com/owncloud/web/issues/6590

## Related Issue
- Fixes https://github.com/owncloud/web/issues/6590

## Screenshots:
Screenshot (11)

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests

## Checklist:

- [x] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

**Review by pascalwengerter — COMMENTED** [review]
Shall we add a config option to disable/enable contextual helpers from an admin's perspective?

**Comment by ownclouders** [comment]
Results for <strong>oC10SharingIntGroupsToRoot</strong> https://drone.owncloud.com/owncloud/web/24797/29/1
<details><summary>:boom: The acceptance tests failed on retry. Please find the screenshots inside ...</summary>

<p>


### webUISharingInternalGroupsToRootEdgeCases-shareWithGroupsEdgeCases_feature-L37.png
  


</p></details>
:boom: The acceptance tests pipeline failed. The build has been cancelled.

**Review by JanAckermann — APPROVED** [review]
NICE!

File: packages/web-app-files/src/components/SideBar/Shares/Collaborators/InviteCollaborator/InviteCollaboratorForm.vue [hunk 1]
```diff
@@ -35,11 +35,15 @@
...
-      class="oc-mt-xs oc-text-meta"
-      v-text="inviteDescriptionMessage"
-    />
+    <p>
+      <span
+        id="files-share-invite-hint"
+        class="oc-mt-xs oc-text-meta"
+        v-text="inviteDescriptionMessage"
+      />
+      <oc-contextual-helper v-if="resourceIsSpace && helpersEnabled" v-bind="spaceAddMemberHelp" />
```
Comment (pascalwengerter) [inline packages/web-app-files/src/components/SideBar/Shares/Collaborators/InviteCollaborator/InviteCollaboratorForm.vue:44]: Would it make sense to reduce this to one OcContextualHelper with only `v-if="helpersEnabled"` and handle the spaces differentiation in the v-bind computed property?

File: yarn.lock [hunk 2]
```diff
@@ -9671,9 +9671,9 @@ __metadata:
   languageName: node
   linkType: hard
 
-"owncloud-design-system@npm:^13.1.0-rc.3":
-  version: 13.1.0-rc.3
-  resolution: "owncloud-design-system@npm:13.1.0-rc.3"
+"owncloud-design-system@npm:^13.1.0-rc.5":
```
Comment (pascalwengerter) [inline yarn.lock:9674]: Should come with a changelog item ;)

**Review by pascalwengerter — CHANGES_REQUESTED** [review]

**Comment by pascalwengerter** [comment]
> can't see the helpers - where have they gone?

Fixed in #7084 by making them opt-out (I guess oCIS never opted in in the first place)

**Pull request merged**